qfieldsync/core/cloud_api.py

```diff
@@ -680,6 +864,32 @@ def _on_login_finished(self, reply: QNetworkReply) -> None:
         )
         self.login_finished.emit()
 
+    def _on_get_user_info_finished(self, reply: QNetworkReply) -> None:
+        try:
+            payload = self.json_object(reply)
+        except CloudException as err:
+            self._login_error = err
+            self.login_finished.emit()
+            self.preferences.set_value("qfieldCloudRememberMe", False)
+            return
+
+        self.user_details = {
+            "username": payload["username"],
+            "avatar_url": payload["avatar_url"],
+        }
+        if payload["avatar_url"]:
+            suffix = payload["avatar_url"].rsplit(".")[-1]
+            avatar_filename = tempfile.mktemp(suffix=f".{suffix}")
+            reply = self.get_file(
+                QUrl(payload["avatar_url"]),
+                avatar_filename,
+            )
+            reply.finished.connect(
+                lambda: self._on_avatar_download_finished(reply, avatar_filename)
+            )
```

Comment:
```suggestion

        if payload["avatar_url"]:
            suffix = payload["avatar_url"].rsplit(".")[-1]
            avatar_filename = tempfile.mktemp(suffix=f".{suffix}")
            reply = self.get_file(
                QUrl(payload["avatar_url"]),
                avatar_filename,
            )
            reply.finished.connect(
                lambda: self._on_avatar_download_finished(reply, avatar_filename)
            )

```

A little bit of air :)